rosdep/python.yaml

```diff
@@ -9863,6 +9863,9 @@ python3-yolov5:
     pip:
       packages: [yolov5]
 python3-zmq:
+  alpine:
+    '*': [py3-pyzmq]
+    '3.8': null
```

Comment:
Do we need to exclude EOL-ed version?

Reply:
An error for Alpine 3.8 occurred in the previous commit: https://github.com/alpine-ros/rosdistro/actions/runs/7666286707/job/20893787054#step:6:1671.

Reply:
Maybe we should modify this list: https://github.com/alpine-ros/rosdistro/blob/a5f67dfadc51a24dbf6dca224ee7c9b5b67f789e/test/rosdep_repo_check/config.yaml#L59-L62

Reply:
As we keep old Alpine ROS docker images, it might be good to keep it